.github/workflows/merge-main.yml

```diff
@@ -291,154 +254,49 @@ jobs:
-        with:
-          scan-type: "fs"
-          format: "sarif"
-          output: "trivy-results.sarif"
-          ignore-unfixed: true
-          severity: "CRITICAL,HIGH"
-          security-checks: "vuln,secret,config"
-
-      - name: Upload Trivy scan results to GitHub Security tab
-        uses: github/codeql-action/upload-sarif@v2
-        with:
-          sarif_file: "trivy-results.sarif"
-
-  image-spar-api:
-    name: Service API Image Handling
-    needs:
-      - zap-spar-api
-    outputs:
-      build: ${{ steps.check.outputs.build }}
-    env:
-      COMPONENT: spar-api
-      PREV: test
-      ZONE: prod
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check for image changes
-        id: check
-        run: |
-          # Vars
-          IMG_PREV="${{ env.REGISTRY }}/${{ github.repository }}:${{ env.PREV }}-${{ env.COMPONENT }}"
-          IMG_ZONE="${{ env.REGISTRY }}/${{ github.repository }}:${{ env.ZONE }}-${{ env.COMPONENT }}"
-          # Pull previous image; grab SHA
-          docker pull "${IMG_PREV}"
-          SHA_PREV=$(docker inspect -f '{{.Id}}' "${IMG_PREV}")
-          # Use blank SHA for promoted image, unless a real one exists instead
-          docker pull "${IMG_ZONE}" && \
-            SHA_ZONE=$(docker inspect -f '{{.Id}}' "${IMG_ZONE}") ||
-            SHA_ZONE=""
-          # Output SHAs
-          echo -e "\n${IMG_PREV}: ${SHA_PREV}"
-          echo -e "${IMG_ZONE}: ${SHA_ZONE}\n"
-          # If different, then trigger updates
-          if [[ "${SHA_PREV}" != "${SHA_ZONE}" ]]; then
-            echo "::set-output name=build::true"
-            echo "Image has changed"
-            exit 0
-          fi
-          echo "Image promotion not required"
-      - name: Promote Service API Image
-        if: steps.check.outputs.build == 'true'
-        uses: shrink/actions-docker-registry-tag@v3
-        with:
-          registry: ${{ env.REGISTRY }}
-          repository: ${{ github.repository }}
-          target: ${{ env.PREV }}-${{ env.COMPONENT }}
-          tags: |
-            ${{ env.ZONE }}-${{ env.COMPONENT }}
-
   deploy-prod:
     name: PROD Deployment
     needs:
-      - image-spar-api
-      - trivy-repo
-      - release
-    runs-on: ubuntu-latest
-    environment:
-      name: prod
+      - api-tests
+    environment: prod
+    runs-on: ubuntu-22.04
     env:
       ZONE: prod
       PREV: test
       NR_SPAR_BACKEND_VERSION: ${{ needs.release.outputs.version }}
+    strategy:
+      matrix:
+        name: [ init, database, backend ]
+        include:
+          - name: init
+            file: common/openshift.init.yml
+            overwrite: false
+          - name: database
+            file: database/openshift.deploy.yml
+            overwrite: false
+          - name: backend
+            file: backend/openshift.deploy.yml
+            overwrite: true
+            parameters: -p MIN_REPLICAS=1 -p MAX_REPLICAS=2 -p BUILD=test-${{ env.NR_SPAR_BACKEND_VERSION }}
```

Comment:
Remove MIN_REPLICAS and MAX_REPLICAS to use the template defaults, which are 3 and 5.

Reply:
Done!